src/runner.cpp

```diff
@@ -20,6 +20,10 @@ void deposit(int &account) {
 }
 
 int main() {
+  LOG.set_filter([](auto &l) {
+    l.info.level += 30;
+    return l.message.str().find("deposit") != std::string::npos;
+  });
```

Comment:
let's document the affect of such a filter on debug line -- won't be excalated to warn if it is not logged in the first place. just that the printed level is editted.